src/string-calculator.ts

```diff
@@ -9,34 +9,74 @@ export class StringCalculator{
 
     public summation(numberList:string): string {
 
-        if(numberList.length=="".length){
+        if(numberList.length==0){
```

Comment:
in JS/TS always use triple eq check '===' this will check the value as well as the type so that 
"alabala"== true will not return true.